src/vm/exceptionhandling.cpp

```diff
@@ -4663,6 +4664,11 @@ VOID DECLSPEC_NORETURN DispatchManagedException(PAL_SEHException& ex)
                     // Unwind to the managed caller of the helper
                     PAL_VirtualUnwind(&frameContext, NULL);
                 }
+#ifdef _TARGET_ARM_
```

Comment:
You mean marked JIT helper does not need PAL_VirtualUnwind ? If we do adjustment directly to the context in both cases, It can be merged.